easybuild/easyconfigs/v/vampir/vampir-9.5.0.eb

```diff
@@ -0,0 +1,69 @@
+easyblock = 'Bundle'
+name = 'vampir'
+version = '9.5.0'
+homepage = 'http://www.vampir.eu'
+description = """ Vampir provides a manageable framework for analysis, which
+enables developers to quickly display program behavior at any level of detail.
+Detailed performance data obtained from a parallel program execution can be
+analyzed with a collection of different performance views.""" 
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+modtclfooter = """
+# HOSTNAME ------------------------------------------------------
+set H        "[exec uname -n]"
```

Comment:
I do not see the use of these variables in the module.
And if you plan to use them, can you please use more descriptive variable name?

Reply:
Good point, thanks. We want to get rid of this modulefile.